.github/ISSUE_TEMPLATE/events.md

```diff
@@ -16,5 +16,6 @@ assignees: ''
 ---
 **Contributor Resources**
 
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

Comment:
I should have noted that e don't need to specify master here. We can just say, "see contributing instructions" without having to specify master.